fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/gradlewbuild.py

```diff
@@ -16,12 +16,15 @@ class GradlewBuild(object):
     def __init__(self, log):
         self.log = log
 
-    def test(self, identifier):
+    def test(self, identifier, smoke=None):
         # self.adbrun.launch()
 
         # Change path accordingly to go to root folder to run gradlew
         os.chdir("../../../../../../../..")
-        cmd = f"adb shell am instrument -w -e class org.mozilla.fenix.experimentintegration.{identifier} org.mozilla.fenix.debug.test/androidx.test.runner.AndroidJUnitRunner"
+        if smoke:
+            cmd = f"adb shell am instrument -w -e class org.mozilla.fenix.ui.{identifier} org.mozilla.fenix.debug.test/androidx.test.runner.AndroidJUnitRunner"    
+        else:
+            cmd = f"adb shell am instrument -w -e class org.mozilla.fenix.experimentintegration.{identifier} org.mozilla.fenix.debug.test/androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Maybe this (with a better parameter name then `cmd_value` 😆)?

```suggestion
        cmd_value = "ui" if smoke else "experimentintegration"
        cmd = f"adb shell am instrument -w -e class org.mozilla.fenix.{cmd_value}.{identifier} org.mozilla.fenix.debug.test/androidx.test.runner.AndroidJUnitRunner"
```